### PR TITLE
[TRANSLATION] Update Simplified Chinese (zh-CN) translation

### PR DESCRIPTION
**Pull request by njlyf2011**
## Purpose

Some improvements about grammar and punctuation.

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-chinese

**Review by eason329 — COMMENTED** [review]
Please check out my review, otherwise LGTM for now.

**Comment by njlyf2011** [comment]
Okay.

File: base/shell/explorer/lang/zh-CN.rc [hunk 1]
```diff
@@ -149,7 +149,7 @@ BEGIN
     LTEXT            "", IDC_STATIC, 7, 51, 258, 1, NOT WS_GROUP
     AUTOCHECKBOX     "不要在启动时执行此检查(&D)", IDC_NOTCHECK, 7, 62, 139, 10
     ICON             32515, IDC_STATIC, 7, 7, 21, 20
-    LTEXT            "有是一个文件或文件夹在您的计算机被称为 %s 这可能会导致某些应用程序不能够正确。重命名为 %s 将解决这个问题。您想现在重命名它?", IDC_STATIC, 34, 7, 224, 37
+    LTEXT            "您的计算机有一个称为 %s 的文件或文件夹，这可能会导致某些应用程序不能够正确执行。将其重命名为 %s 则可以解决这个问题。您想现在重命名它？", IDC_STATIC, 34, 7, 224, 37
```
Comment (kernelbin) [inline base/shell/explorer/lang/zh-CN.rc:152]:
```suggestion
    LTEXT            "您的计算机内有一个称为 %s 的文件或文件夹，这可能会导致某些应用程序不能够正确执行。将其重命名为 %s 则可以解决这个问题。您想现在重命名它？", IDC_STATIC, 34, 7, 224, 37
```

Comment (kernelbin) [reply base/shell/explorer/lang/zh-CN.rc:152]: does this sounds better?

**Review by kernelbin — APPROVED** [review]

**Comment by kernelbin** [comment]
LGTM!

**Review by eason329 — APPROVED** [review]

File: base/applications/utilman/lang/zh-CN.rc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/*
+ * PROJECT:         ReactOS Utility Manager (Accessibility)
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Chinese (Simplified) translation resource
+ * TRANSLATOR:      Copyright 2022 Luo Yufan <njlyf2011@hotmail.com>
+ */
```
Comment (binarymaster) [inline base/applications/utilman/lang/zh-CN.rc:6]:
```suggestion
/*
 * PROJECT:     ReactOS Utility Manager (Accessibility)
 * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
 * PURPOSE:     Chinese (Simplified) resource file
 * TRANSLATOR:  Copyright 2022 Luo Yufan <njlyf2011@hotmail.com>
 */
```

File: base/applications/notepad/lang/zh-CN.rc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/* Translation has been changed by Luo Yufan on 2018-12-05 (njlyf2011@hotmail.com) */
+/* Translation has been updated by Luo Yufan on 2022-01-03 (njlyf2011@hotmail.com) */
```
Comment (binarymaster) [inline base/applications/notepad/lang/zh-CN.rc:1]:
```suggestion
/*
 * PROJECT:     ReactOS Notepad
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Chinese (Simplified) resource file
 * TRANSLATOR:  Copyright 2018, 2022 Luo Yufan <njlyf2011@hotmail.com>
 */
```

**Review by binarymaster — APPROVED** [review]
LGTM, just a few headers to fix.

**Pull request merged**